documentation/docs/pmm-upgrade/upgrade_docker.md

```diff
@@ -10,7 +10,7 @@ Before starting the upgrade, complete these preparation steps to ensure you can
 2. Verify your current PMM version: Check your current PMM version by navigating to **PMM Configuration > Updates** or by running the following command: 
```

Comment:
Since we have backup data as second step, should we have it in "Before you begin"? I think we can remove it.